utils/ansible_playbook_to_role.py

```diff
@@ -252,6 +270,36 @@ def _description(self):
                 desc += (line + "\n")
         return desc.strip("\n\n")
 
+    @property
+    def _update_galaxy_tags(self):
+        galaxy_tags = {}
+        # These are the default tags that all roles share
+        tags = [
+            "system",
+            "hardening",
+            "openscap",
+            "ssg",
+            "scap",
+            "security",
+            "compliance",
+            "complianceascode",
+            "redhatofficial",
+            "redhat",
+        ]
+        prod, prof = self.name.split("_", 1)
```

Comment:
Just a comment that it may be worthy storing `product` and `profile` as a property, or something, of the class for easy access.